tardis/adapters/sites/kubernetes.py

```diff
@@ -1,4 +1,5 @@
 from kubernetes_asyncio import client as k8s_client
+from kubernetes_asyncio import client as hpa_client
```

Comment:
Is there any reason why you re-import ``kubernetes_asyncio.client`` just with another alias?
It seems you are just using the same client with different configuration – it's fine to reuse the  single *global* ``k8s_client`` type/function, the properties on ``KubernetesAdapter`` make the distinction when it is appropriate.

Reply:
only the k8s_client is now used